src/kv/CMakeLists.txt

```diff
@@ -12,13 +12,3 @@ add_library(kv STATIC $<TARGET_OBJECTS:kv_objs>)
 target_include_directories(kv_objs SYSTEM BEFORE PUBLIC ${ROCKSDB_INCLUDE_DIR})
 target_include_directories(kv SYSTEM BEFORE PUBLIC ${ROCKSDB_INCLUDE_DIR})
 target_link_libraries(kv ${LEVELDB_LIBRARIES} ${ROCKSDB_LIBRARIES} ${ALLOC_LIBS} ${SNAPPY_LIBRARIES} ${ZLIB_LIBRARIES})
-
-# rocksdb detects bzlib and lz4 in its Makefile, which forces us to do the same.
```

Comment:
these changes were added to accommodate the Makefile of rocksdb which predates CMakefile of it.